packages/ihe-gateway-sdk/src/models/patient-discovery.ts

```diff
@@ -0,0 +1,30 @@
+import { Patient } from "@medplum/fhirtypes";
+import { NPIStringArray, oidStringSchema, SamlAttributes, baseResponseSchema } from "./shared";
+import { z } from "zod";
+
+export const xcpdGatewaysSchema = z.array(
+  z.object({
+    oid: oidStringSchema,
+    url: z.string(),
+    id: z.string(),
+  })
+);
+
+export type XCPDGateways = z.infer<typeof xcpdGatewaysSchema>;
+
+export type PatientDiscoveryRequest = {
+  id: string;
+  cxId: string;
+  timestamp: string;
+  xcpdGateways: XCPDGateways;
```

Comment:
nit: does it need to state `xcpdGateways`, couldn't it just say `gateways`?

Reply:
Yea this is how we have it defined in the Mirth channels